src/components/GameInfoV2/index.tsx

```diff
@@ -0,0 +1,191 @@
+  socialLinks,
+  editorChoice,
+  actionButton,
+  isLoading,
+  className
+}): JSX.Element => {
+  const [isImageLoading, setIsImageLoading] = useState(isLoading)
+
+  useEffect(() => {
+    if (ImageComponent && !isImageLoading) {
+      setIsImageLoading(false)
+    }
+    if (!ImageComponent && isImageLoading) {
+      setIsImageLoading(true)
+    }
+  }, [ImageComponent, isImageLoading])
+
+  let editorChoiceElement = null
+  if (editorChoice?.isEditorChoice) {
+    editorChoiceElement = (
+      <div className={classNames(styles.editorChoice, editorChoice.className)}>
+        <EditorChoice />
+        Editor&apos;s Choice {editorChoice.year || new Date().getFullYear()}
+      </div>
+    )
+  }
+
+  const socialIcons = (socialLink: string) => {
+    if (socialLink === 'website') {
+      return {
+        icon: Globe,
+        className: styles.websiteIcon
+      }
+    }
+    if (socialLink === 'twitter') {
+      return {
+        icon: X,
+        className: styles.twitterIcon
+      }
+    }
+    if (socialLink === 'discord') {
+      return {
+        icon: DiscordFilled,
+        className: styles.discordIcon
+      }
+    }
+    if (socialLink === 'youtube') {
+      return {
+        icon: Youtube,
+        className: styles.youtubeIcon
+      }
+    }
+  }
+
+  return (
+    <div className={classNames(styles.container, className)}>
+      <div className={styles.imageContainer}>
+        {isLoading ? <div className={styles.imageLoading} /> : ImageComponent}
+      </div>
+
+      <div className={styles.body}>
+        <div className={styles.content}>
+          <div className={styles.header}>
+            {editorChoiceElement}
+            <span className={classNames(styles.title)}>{title}</span>
+          </div>
+
+          <div className={styles.info}>
+            <div className={styles.badges}>
+              <MetaSection
+                title=""
+                items={[
+                  <Sticker
+                    key="version"
+                    styleType="neutral"
+                    variant="filledStrong"
+                  >
+                    {version}
+                  </Sticker>,
+                  <Sticker
+                    key="earlyAccess"
+                    styleType="neutral"
+                    variant="filledStrong"
+                  >
+                    {earlyAccess ? 'Early Access' : ''}
+                  </Sticker>,
+                  <Sticker
+                    key="developer"
+                    styleType="neutral"
+                    variant="filledStrong"
+                  >
+                    {info.developer}
+                  </Sticker>,
+                  <Sticker
+                    key="playerCount"
+                    styleType="neutral"
+                    variant="filledStrong"
+                  >
+                    {playerCount}
+                  </Sticker>
+                ]}
+                classNames={{}}
+                maxVisibleItems={5}
```

Comment:
I think we should make this something like 20. The popover for the meta section component still needs some styling tweaks and design/prod mentioned to me that we should just show all the tags for now